eng/scale-set-extension-scripts/startup.ps1

```diff
@@ -0,0 +1,9 @@
+param(
+  [Parameter(Mandatory = $True)]
+  [string]
+  $AppInsightsConnectionString
+)
+
+setx APPLICATION_INSIGHTS_CONNECTION_STRING "$AppInsightsConnectionString" /M
```

Comment:
Is this the standard way we are doing this or something new?

Reply:
this is the standard way, we set an environmental variable in this Extension that gets picked up by the service when it's initializing